e2e/self_node_remediation_test.go

```diff
@@ -75,11 +75,12 @@ var _ = Describe("Self Node Remediation E2E", func() {
 
 		AfterEach(func() {
 			// restart snr pods for resetting logs...
-			restartSnrPods(workers)
+			//restartSnrPods(workers)
 		})
 
 		JustAfterEach(func() {
-			printSNRLogsFromNode(&workers.Items[1])
+			// TODO why worker 1??
```

Comment:
No idea, it must be a mistake

Reply:
ah no, it comes from a "refactory", originally it was the log of `healthyNode := &workers.Items[1]`

Reply:
I could've kept at least the variable name to make it clearer